docs/partners/become-a-tech-partner.md

```diff
@@ -118,3 +118,10 @@ Now that the integration is complete, you'll get the visibility you deserve! The
 
 Partners who wish to contribute paid packs must follow a few [additional onboarding steps](/docs/partners/premium-packs-process). 
 
+## 8. Suggested Timeline
```

Comment:
@edik24 I think this will read better if its included in section 5 as part of development, not its own section.

Reply:
Moved it to section 5 as suggested